nebula_ros/src/continental/continental_srr520_ros_wrapper.cpp

```diff
@@ -0,0 +1,288 @@
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<double>("receiver_timeout_sec", descriptor);
+    config.receiver_timeout_sec = this->get_parameter("receiver_timeout_sec").as_double();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_only();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_DOUBLE;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<double>("sender_timeout_sec", descriptor);
+    config.sender_timeout_sec = this->get_parameter("sender_timeout_sec").as_double();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_only();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("filters", descriptor);
+    config.filters = this->get_parameter("filters").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+    descriptor.read_only = false;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("frame_id", descriptor);
+    config.frame_id = this->get_parameter("frame_id").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_STRING;
+    descriptor.read_only = false;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<std::string>("base_frame", descriptor);
+    config.base_frame = this->get_parameter("base_frame").as_string();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_BOOL;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<bool>("use_bus_time", descriptor);
+    config.use_bus_time = this->get_parameter("use_bus_time").as_bool();
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_DOUBLE;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<double>("configuration_vehicle_wheelbase", descriptor);
+    config.configuration_vehicle_wheelbase =
+      static_cast<float>(this->get_parameter("configuration_vehicle_wheelbase").as_double());
+  }
```

Comment:
Please adopt the new, shorter style of defining parameters:
* `descriptor.type` is ignored when calling `declare_parameter<type>` so it can be omitted
* `descriptor.read_only` can be omitted as `param_read_write` and `param_read_only` are used
* `descriptor.dynamic_typing` is `false` by default and can be omitted
* `descriptor.additional_constraints` is `""` by default and can be omitted
* `declare_parameter()` followed directly by `x = get_parameter()` should be replaced by `x = declare_parameter()`

For most parameters, the code should thus become: `config.x = declare_parameter<int>("x", param_read_only());` without any `{}` block scope and explicit `descriptor`.

Reply:
Addressed in [78996f6](https://github.com/tier4/nebula/pull/124/commits/78996f6fd4a346213afc1c5a90fb5155ea61a68b)